kubeflow/core/jupyterhub_spawner.py

```diff
@@ -5,10 +5,16 @@
 from oauthenticator.github import GitHubOAuthenticator
 
 class KubeFormSpawner(KubeSpawner):
+
+  # relies on HTML5 for image datalist
   def _options_form_default(self):
     return '''
     <label for='image'>Image</label>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-    <input name='image' placeholder='repo/image:tag'></input>
+    <input list="image" name="image" placeholder='repo/image:tag'>
+    <datalist id="image">
+      <option value="gcr.io/kubeflow/tensorflow-notebook-cpu">
```

Comment:
The latest images are in
gcr.io/kubeflow-images-staging/tensorflow-notebook-cpu
gcr.io/kubeflow-images-staging/tensorflow-notebook-gpu

Its part of our evolving release process.